src/plugins/intel_cpu/src/nodes/input.cpp

```diff
@@ -350,10 +351,19 @@ void Input::cloneBlobIfRequired() {
                 + "_" + ptr;
     };
 
+
+    bool needFlushDenormalsToZero;
+    if (is_denormals_as_zero_set()) {
+        // CPU flag DAZ has been set, no need to flush denormals to zero
+        needFlushDenormalsToZero = false;
+    } else {
+        needFlushDenormalsToZero = hasSubnormals();
+    }
+
     if (weightCache) {
         MemoryPtr ptr = *weightCache->findOrCreate(blobKey(), cloneBlob);
         memoryPtr = std::const_pointer_cast<const Memory>(ptr);
-    } else if (isBlobAligned() && !hasSubnormals() && !isWA()) {
+    } else if (isBlobAligned() && !needFlushDenormalsToZero && !isWA()) {
```

Comment:
Also since the data can be reallocated not only because of the sumbnormal numbers, please add the intriduced DAZ check into `cloneBlob()` call, specifically pass the DAZ check result as a second parameter to 'SetData' call, so that we save some graph loading time.

Reply:
Good idea, will change